plutus-e2e-tests/test/Helpers.hs

```diff
@@ -39,10 +39,31 @@ import Test.Runtime qualified as TN
 import Testnet.Conf qualified as TC (Conf (..), ProjectBase (ProjectBase), YamlFilePath (YamlFilePath), mkConf)
 import Testnet.Plutus qualified as TN
 
-testnetOptionsAlonzo6, testnetOptionsBabbage7, testnetOptionsBabbage8 :: TN.TestnetOptions
-testnetOptionsAlonzo6   = TN.defaultTestnetOptions {TN.era = C.AnyCardanoEra C.AlonzoEra,  TN.protocolVersion = 6}
-testnetOptionsBabbage7 = TN.defaultTestnetOptions {TN.era = C.AnyCardanoEra C.BabbageEra, TN.protocolVersion = 7}
-testnetOptionsBabbage8 = TN.defaultTestnetOptions {TN.era = C.AnyCardanoEra C.BabbageEra, TN.protocolVersion = 8}
+data LocalNodeOptions = LocalNodeOptions
+  { eraL             :: C.AnyCardanoEra
+  , protocolVersionL :: Int
```

Comment:
What does the L mean in `eraL` and `protocolVersionL`?

Reply:
"local" (or "left"). it doesn't need to be different to cardano-testnet's `era` if I rearrange the imports. Do plan on separating the testnet helpers and other types of helper functions.  If it's cleaner, I can change this to `era'` for now?

Reply:
scratch that, was true before moving things around, changed to `era`

Reply:
Ah, right! In cardano-ledger there is now a lens interface and all the lens have an L at the end, that got me confused.

Reply:
Ok, definitely won't append `L` in future then :)